go/vt/vtgate/evalengine/compiler_test.go

```diff
@@ -340,6 +340,10 @@ func TestCompilerSingle(t *testing.T) {
 			expression: `WEEK(date '2014-10-26', 6)`,
 			result:     `INT64(44)`,
 		},
+		{
+			expression: `MAKETIME(0, 0, 0x0)`,
+			result:     `TIME("00:00:00")`,
+		},
```

Comment:
How about adding a `MAKEDATE` test too?

Reply:
@frouioui We already have a whole slew of those. In fact, I probably should drop this one here since it's not useful as we have the generated tests already :smile:.